app/components/Settings/cron-picker/cron-helpers.js

```diff
@@ -10,6 +10,40 @@ export function mapHourToCronFormat(hour, amPm) {
     return hour;
 }
 
+export function mapCronToHourFormat(cronExpression) {
+    if (!cronExpression) {
```

Comment:
How about validating the cron expressions using cronstrue (untested)?
```js
try {
    cronstrue.toString(cronExpression);
} catch (_) {
    return null;
}

Reply:
awesome idea! I added that in